src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1300,7 +1300,7 @@ internal static bool IsSpanType(this TypeSymbol type)
             }
 
             // the "System" must be in the global namespace
-            return ns.ContainingNamespace.IsGlobalNamespace;
+            return ns.ContainingNamespace?.IsGlobalNamespace == true;
```

Comment:
Why did we need this change?  I can't see how it's related to the other changes here?